week-3/Homework/mandatory/5-journey-planner.js

```diff
@@ -7,16 +7,38 @@
   NOTE: only the names should be returned, not the means of transport.
 */
 
-function journeyPlanner() {
 
+function journeyPlanner(transportMode){
+  const londonLocations = [
+  ["Angel", "tube", "bus"],
+  ["London Bridge", "tube", "river boat"],
+  ["Tower bridge", "tube", "bus"],
+  ["Greenwich", "bus", "river boat"]
+]
+  let bus =[];
+  let tube =[];
+  let riverBoat =[];
+  if(transportMode == "bus"){
```

Comment:
ooooor you can do it in one line

function journeyPlanner(londonLocations, transport) {
return londonLocations.filter(location => location.includes(transport)).map(location => location = location[0]);
   
  
}